lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -14,13 +14,11 @@ class Results
 
         # Find the latest Abbyy results XML file by druid, since there could be more than one.
         # @param druid [String] The druid to use to look for results
-        # @param logger [Logger] An optional logger to use
-        # @return [Results, nil] The Results object or nil, if one wasn't found.
-        def self.find_latest(druid:, logger: nil)
+        # @return [xml_filepath, nil] The result xml file path or nil, if one wasn't found.
+        def self.find_latest(druid:)
           bare_druid = DruidTools::Druid.new(druid).id
           # NOTE: Dir.glob will sort the filenames in ascending order and we want the last one.
-          result_files = Dir.glob("#{Settings.sdr.abbyy.local_result_path}/#{bare_druid}*.xml")
-          Results.new(result_path: result_files.last, logger:) unless result_files.empty?
+          Dir.glob("#{Settings.sdr.abbyy.local_result_path}/#{bare_druid}*.xml").last
```

Comment:
this method now just finds the file, it does not initialize the result object and thus try to parse the XML